sample_project/Assets/SampleViewer/Samples/MaterialByAttribute/MaterialByAttribute.unity

```diff
@@ -1973,7 +2001,7 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 44f3381abd4f52e48928147257044361, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  APIKey: AAPK1733da5491544f05ae71518d08e30ea5RW7FclTJA-x6-LAshjwcCYEUVMwB41TVhmd6_BLXpj5xcgDkYuzACtVxhwpGl-oZ
```

Comment:
This commit will need to be force pushed to remove the API key from commit history

Reply:
Thats fine, better late than never. We will be forced to rewrite this commit at some point no matter what though since it should show up in a repo scan.